src/translations/en.json

```diff
@@ -975,7 +975,7 @@
             "description": "Trigger automations when a NFC tag, QR code, etc. is scanned"
           },
           "people": {
-            "title": "People & Zones",
+            "title": "People, Zones & Users",
             "description": "Manage the people and zones that Home Assistant tracks"
```

Comment:
```suggestion
            "title": "People & Zones",
            "description": "Manage the people, zones, and users that Home Assistant tracks"
```

Can we put it in the description instead?

Reply:
As I wrote above, that would in some/many languages result in cut off descriptions (shown with ellipsis) since it will get too long. E.g. in German basically all words are longer than their English counterparts and there is not much space left today:

![image](https://user-images.githubusercontent.com/114137/149236192-4a5416ca-e814-4a2a-96ff-165fe4436581.png)